apps/codeimage/src/pages/Dashboard/dashboard.state.ts

```diff
@@ -27,10 +27,12 @@ function makeDashboardState(authState = getAuth0State()) {
       );
     return [];
   };
-
-  const [pagedData, {page, setPage, pageSize}] = createPagedData(() =>
-    filteredData(),
-  );
+  const pageSize = 9;
+  const filteredDataAccessor = () => filteredData();
+  const [pagedData, {page, setPage}] = createPagedData({
```

Comment:
for a question of "project best practices" the accessor should be the first argument, then the options should be handled in the second argument